src/hotspot/share/gc/shenandoah/shenandoahCodeRoots.cpp

```diff
@@ -34,6 +34,7 @@
 #include "memory/resourceArea.hpp"
 #include "memory/universe.hpp"
 #include "runtime/atomicAccess.hpp"
+#include "runtime/icache.hpp"
```

Comment:
Include is added, but no actual use? Is something missing, or this is a leftover include?